### DISCUSSION TITLE
Auth module incompatibility with Github Codespaces

### DISCUSSION THREAD

**Discussion by DiesIrae** [keystonejs/keystone#8394] [Questions] [answered]
### To reproduce:
1. Go to https://github.com/keystonejs/keystone
2. Code > Crete a Codespace on main
3. Run `yarn install`, then `yarn run dev` in `/examples/auth`
4. Ctrl+click on the localhost link
5. Setup new user
6. Sign out
7. `400 - bad request` Impossible to sign in

### Context

I am trying to setup a development environment for Keystone in [Github Codespaces](https://docs.github.com/en/codespaces). Everything works fine except authentication. When I access `/signin` I get a `400 - bad request` and no error in Keystone logs. But `/hello` & `/api/graphql` work, and the admin ui works fine if I remove the auth or go to no-auth examples.

When trying to debug I added logs in express via [extendExpressApp](https://keystonejs.com/docs/config/config#extend-express-app), saw that only `/favicon` passes through.

Could you help me? Did anyone try Keystone + GIthub Codespaces before?

**Comment by DiesIrae** [comment, marked as answer]
Ok I just found it's a Codespaces problem, sorry for the disturbance https://github.com/orgs/community/discussions/5204

**Comment by DiesIrae** [comment]
Could the `/signin` uri be rendered customizable? It won't be fixed soon on their part it seems.

**Reply by dcousens** [reply]
@DiesIrae this isn't too many places you'd need to modify in https://github.com/keystonejs/keystone/blob/main/packages/auth/src/index.ts, but I don't know if we want to add features specifically to work around this.

**Reply by DiesIrae** [reply]
@dcousens thanks for your reply! Yeah I understand... Sad though that they don't consider it a big problem, Codespaces is great for occasional contributors.
